railway-deploy.sh

```
@@ -18,17 +18,54 @@ railway link
 
 echo "🔧 Setting environment variables..."
 
+# Load from .env if exists
+if [ -f .env ]; then
+    export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)